src/DocBlock/Line.php

```diff
@@ -42,6 +42,8 @@ public function __construct($content)
      * Get the string representation of object.
      *
      * @return string
+     *
+     * @deprecated will be removed in 3.0, use getContent
```

Comment:
Shouldn't the method make a `@trigger_error('...', E_USER_DEPRECATION)` call?

Reply:
risky, because a lot of IDE's call this method to show preview data when debugging

Reply:
Wouldn't `@` make it transparent for IDEs?